searx/webapp.py

```diff
@@ -1329,6 +1332,9 @@ def page_not_found(_e):
     return render('404.html'), 404
 
 
+userdoc.render(app)
```

Comment:
Nit: the userdoc is initialized twice in debug mode. To avoid that, and to group the initialization code in place, can you move this call just after `search_initialize(...)` (line 1352).